### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by khdimon**
Let the documentation be stored in the master.

### CODE REVIEW

File: doc/LOGIN.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+Required: `username=[String], password=[String]`
+
+Example: `username=OlegShvets, password=ghd22df`
+
+**Data type :** form-data
+
+**Success Response:** Code: 200
+
+**Error Response :** Code: 401 Unauthorized, 405 Method Not Allowed
```
Comment (thedikiy) [inline doc/LOGIN.md]: Login has no 401 response code

File: src/main/resources/init.sql [hunk 2]
```diff
@@ -2,10 +2,10 @@ DROP SCHEMA IF EXISTS caesar;
 CREATE SCHEMA caesar CHARACTER SET UTF8;
 USE caesar;
 
-CREATE USER IF NOT EXISTS 'db_admin'@'localhost' IDENTIFIED BY 'rNT5Vn[j(>R2r6.UXcJwtv];`RWewYH`';
+CREATE USER 'db_admin'@'localhost' IDENTIFIED BY 'rNT5Vn[j(>R2r6.UXcJwtv];`RWewYH`';
 GRANT ALL PRIVILEGES ON caesar.* TO 'db_admin'@'localhost';
 
-CREATE USER IF NOT EXISTS 'user_ro'@'localhost' IDENTIFIED BY '12345qwerty';
+CREATE USER 'user_ro'@'localhost' IDENTIFIED BY '12345qwerty';
 GRANT SELECT ON caesar.* TO 'user_ro'@'localhost';
```
Comment (thedikiy) [inline src/main/resources/init.sql]: Do we need script changes in documentation branch?

**Comment by svoronin** [comment]
That's a great documentation! Superb work, guys!

File: doc/README.MD [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# API Documentation
+
+This is documentation for project API. It includes:
+
+- [LOGIN.md](LOGIN.md)
+- [LOGOUT.md](LOGOUT.md)
+- [USERS.md](USERS.md)
+- [USERPROFILE.md](USERPROFILE.md)
+- ...
```
Comment (svoronin) [inline doc/README.MD]: No need for those dots.

File: doc/LOGIN.md (hunk 1 above)
Comment (khdimon) [inline doc/LOGIN.md]: When I send wrong credentials I have this responce code

Comment (thedikiy) [inline doc/LOGIN.md]: Ah, okay you are right

**Pull request merged**